recipes-extended/xen/files/libxl-openxt-tweaks.patch

```diff
@@ -0,0 +1,133 @@
+################################################################################
+SHORT DESCRIPTION:
+################################################################################
+LibXL modifications specific to OpenXT
```

Comment:
Like which ones? The other one that's kinda "misc" is the RFC fixes one, but it's not OpenXT-specific so it doesn't belong with these changes.
The idea is to keep upstreamable stuffs separate from non-upstreamable stuffs.